recipe/build.sh

```diff
@@ -30,10 +30,21 @@ fi
     --arch="${PYTHIA_ARCH}" \
     --prefix="${PREFIX}" \
     --with-lhapdf6="${PREFIX}" \
+    --with-fastjet3="${PREFIX}" \
     --with-gzip="${PREFIX}" \
     --with-mg5mes
 
 make install --jobs="${CPU_COUNT}"
+make clean
+
+# Remove documentation and examples from share/Pythia8 as not needed for runtime.
+# Keep share/Pythia8/examples/Makefile.inc as that might be looked for.
+# Keep share/Pythia8/xmldoc as required at runtime.
+mv $PREFIX/share/Pythia8/examples/Makefile.inc .
+rm -rf $PREFIX/share/Pythia8/examples/*
+mv Makefile.inc $PREFIX/share/Pythia8/examples/Makefile.inc
+rm -rf $PREFIX/share/Pythia8/htmldoc
+rm -rf $PREFIX/share/Pythia8/pdfdoc
```

Comment:
@ph-ilten if you're fine with this, it drops the distributed package size from 40 MB to 20 MB, which is good all around.